aws/cloud-formation/codeArtifact/README.md

```diff
@@ -0,0 +1,98 @@
+# CodeArtifact Setup Guide
+
+## Introduction
+
+The Viite project uses AWS CodeArtifact to store and manage our npm and Maven packages. 
+
+**Note for Local Development:**
+While CodeArtifact is used for our CI/CD pipeline, its use is optional for local development. You can choose to develop using public package repositories if you prefer. 
+
+This guide explains how to set up CodeArtifact for both npm (frontend) and SBT/Maven (backend) dependencies, should you choose to use it for your local development.
+
+## Prerequisites
+
+- AWS CLI installed and configured
+- Node.js and npm installed (for frontend)
+- SBT installed (for backend)
+
+## AWS SSO Login
+
+Before setting up CodeArtifact, log in to AWS SSO:
+
+```bash
+aws sso login --profile <profilename>
+```
+Replace <profilename> with your AWS profile name.
+
+## CodeArtifact npm Setup 
+
+Log in to the npm repository:
```

Comment:
npm on  meidän CodeArtifactissa olevan repon nimi, onko?
->
Log in to our npm repository:
or
Log in to the npm repository in our CodeArtifact:

or something like that